src/main/java/com/faforever/client/clan/ClanTooltipController.java

```diff
@@ -20,16 +20,19 @@
   public Label descriptionLabel;
   public Label membersLabel;
   public Label leaderLabel;
+  public Label descriptionTitleLabel;
 
   public void initialize() {
     descriptionLabel.managedProperty().bind(descriptionLabel.visibleProperty());
+    descriptionTitleLabel.managedProperty().bind(descriptionTitleLabel.visibleProperty());
     descriptionLabel.visibleProperty().bind(descriptionLabel.textProperty().isNotEmpty());
+    descriptionTitleLabel.visibleProperty().bind(descriptionLabel.textProperty().isNotEmpty());
   }
 
   public void setClan(Clan clan) {
     nameLabel.setText(clan.getName());
     // TODO improve formatting
-    membersLabel.setText(clan.getMembers().toString());
+    membersLabel.setText(String.valueOf(clan.getMembers().size()));
```

Comment:
`String.valueOf()` shouldn't be used to display integers since it doesn't consider the user's locale. Use `i18n.number()` instead.